discord/interpreter.js

```diff
@@ -213,7 +213,8 @@ module.exports.handleCom = async (message, client) => {
                 } else {
                     Logger.log("info", "Admin channel found. Sending message...");
                     channel.send(`New user ${message.author.username} (real name apparently ${realName}, fresher: ${fresherStr}, identification provided: ${identification}) has requested to join the server.
-If the user seems legitimate, please add them via \`!insertuser ${message.author.username}, ${fresher ? "fresher" : "non-fresher"}, ${realName}\``);
+If the user seems legitimate, please add them via:`);
+                    channel.send(`!insertuser ${message.author.username}, ${fresher ? "fresher" : "non-fresher"}, ${realName}`);

```

Comment:
To keep this part of the message formatted as code in Discord we should add escape backticks. I personally think it makes it easier to see. .e.g.

<code>
channel.send(`\`!insertuser ${message.author.username}, ${fresher ? "fresher" : "non-fresher"}, ${realName}\``);
</code>